File: Tarea8/Libro.java
import java.util.Random;
import java.util.Scanner;

public class Libro {


    private int id;
    private String name;
    private String autor;

    private int cantidad;

    public Libro() {

    }
    public Libro(String name, String autor) {
        this.name = name;
        this.autor = autor;
    }
    public Libro(String name, String autor, int id, int cantidad) {
        this.name = name;
        this.autor = autor;
        this.id = id;
        this.cantidad = cantidad;
    }

    public int getId() {
        return id;
    }

    public void setId() {
        Random random = new Random();
        this.id = random.nextInt(100) + 1;
    }

    public String getName() {
        return name;
    }
    public void setName() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Ingresa el nombre del libro: ");
        this.name = scanner.nextLine();
    }
    public String getAutor() {
        return autor;
    }
    public void setAutor() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Ingresa el nombre del autor");
        this.autor = scanner.nextLine();
    }

    public int getCantidad() {
        return cantidad;
    }

    public void setCantidad() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Ingrese la cantidad disponible: ");
        this.cantidad = scanner.nextInt();
    }
    public void rentarLibro() {
                if (cantidad - 1 >= 0) {
                    cantidad -= 1;
                    System.out.println("Cantidad restante: " + cantidad);
                } else if (cantidad -1 < 0) {
                System.out.println("El libro no se encuentra disponible\n");
            }
        }
}

File: Tarea8Python/Libro.py
import random

class Libro:
    def __init__(self, name=None, autor=None, id=None, cantidad=None):
        self.id = id
        self.name = name
        self.autor = autor
        self.cantidad = cantidad

    def get_id(self):
        return self.id

    def set_id(self):
        self.id = random.randint(1, 100)

    def get_name(self):
        return self.name

    def set_name(self):
        self.name = input("Ingresa el nombre del libro: ")

    def get_autor(self):
        return self.autor

    def set_autor(self):
        self.autor = input("Ingresa el nombre del autor: ")

    def get_cantidad(self):
        return self.cantidad

    def set_cantidad(self):
        self.cantidad = int(input("Ingrese la cantidad disponible: "))

    def rentar_libro(self):
        if self.cantidad - 1 >= 0:
            self.cantidad -= 1
            print("Cantidad restante:", self.cantidad)
        else:
            print("El libro no se encuentra disponible\n")
